sympy/logic/boolalg.py

```diff
@@ -994,6 +1061,40 @@ def _distribute(info):
         return info[0]
 
 
+def to_anf(expr, complete=True):
+    """
+    Converts expr to Algebraic Normal Form (ANF).
+
+    ANF is a canonical normal form, which means that two
+    equivalent formulas will convert to the same ANF.
+
+    A logical expression is in ANF if it has the form
+    ((A & B & ...) ^ (B & C ...) ^ B ^  True ^ ...),
+    i.e. it is purely true, purely false or exclusive disjunction.
+    The exclusive disjunction can only contain true, variables or
+    conjunction of variables. No negations are permitted.
+
+    If ``complete`` is false, arguments of the boolean
+    expression considered as variables.
```

Comment:
I'm confused what this means. Does it mean that if `complete=False`, then only the top-level expression is converted to ANF? If so, a more standard SymPy name for this is `deep=False`.